### PR TITLE
added babyweights keras dnn lab preprocessing layer version

### PR DESCRIPTION
**Pull request by takumiohym**
Sub task of #194

### CODE REVIEW

File: notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb [hunk 1]
```diff
@@ -11,7 +11,6 @@
     "1. Set CSV Columns, label column, and column defaults\n",
```
Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <ul><li>Add back a "Lab Task" mention or "Exercise" mention to help locate the exercises at a glance</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <ul><li>Add back a "Lab Task" mention or "Exercise" mention to help locate the exercises at a glance</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <ul><li>Add back a "Lab Task" mention or "Exercise" mention to help locate the exercises at a glance</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <ul><li>Add back a "Lab Task" mention or "Exercise" mention to help locate the exercises at a glance</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <p>Same remarks as in the solution:</p><ul><li>Replace StringLookup by CategoryEncoding</li><li>Replace "cc" by a more descriptive name</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <ul><li>Add back a "Lab Task" mention or "Exercise" mention to help locate the exercises at a glance</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <ul><li>Add back a "Lab Task" mention or "Exercise" mention to help locate the exercises at a glance</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <ul><li>Add back a "Lab Task" mention or "Exercise" mention to help locate the exercises at a glance</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <ul><li>Add back a "Lab Task" mention or "Exercise" mention to help locate the exercises at a glance</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

**Review by BenoitDherin — APPROVED** [review]
LGTM. Great work!

File: notebooks/end-to-end-structured/solutions/4b_keras_dnn_babyweight.ipynb [hunk 2]
```diff
@@ -11,7 +11,6 @@
     "1. Set CSV Columns, label column, and column defaults\n",
```
Comment (sanjanalreddy) [inline notebooks/end-to-end-structured/solutions/4b_keras_dnn_babyweight.ipynb:11]: <pre class="ql-syntax" spellcheck="false">Line #9.        for nc in NUMERICAL_COLUMNS:
</pre><p>let's change nc here to numerical_column </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (sanjanalreddy) [inline notebooks/end-to-end-structured/solutions/4b_keras_dnn_babyweight.ipynb:11]: <pre class="ql-syntax" spellcheck="false">Line #9.        # Create two hidden layers of [64, 32] just in like the BQML DNN
</pre><p>remove mention of BQML DNN</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

**Review by sanjanalreddy — APPROVED** [review]
LGTM! Minor suggestions.

File: notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb (hunk 1 above)
Comment (takumiohym) [reply notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb:11]: <p>Good catch! Thank you!</p>

**Pull request merged**